migration/sql-files/051-identity-deactivation-scheduled.sql

```diff
@@ -0,0 +1,2 @@
+-- Add a column to track when deactivation has been scheduled for an identity
+ALTER TABLE identities ADD COLUMN deactivation_scheduled timestamp with time zone;
```

Comment:
Will it help if we create an index for that field?

Reply:
Possibly.. I'll add one.